style.css

```diff
@@ -262,4 +258,122 @@
     border-bottom: 1px solid #dfe1e6;
     padding-bottom: 20px;
   }
+
+  /* contact me */
+  .contact-me {
+    width: 375px;
+    height: 779px;
+    position: absolute;
+    margin-top: -160px;
+    border-top-left-radius: 100px;
+    background-color: #6070ff;
+  }
```

Comment:
- [ ] In order to layout the elements on the page you should use Flexbox as required, please add `display: flex `on this section

Reply:
Updated accordingly. Thanks